neuralmonkey/attention/base_attention.py

```diff
@@ -50,13 +51,15 @@ def empty_attention_loop_state() -> AttentionLoopStateTA:
     two empty arrays, one for attention distributions in time, and one for
     the attention context vectors in time.
     """
-    return AttentionLoopStateTA(
-        contexts=tf.TensorArray(
-            dtype=tf.float32, size=0, dynamic_size=True,
+    return AttentionLoopState(
+        contexts=tf.zeros(
+            shape=[0, batch_size],
+            dtype=tf.float32,
```

Comment:
Neni to humus dělat matice s dimenzí **nula**, něco?

Reply:
rozmyslel jsem si to a zas takovej humus to neni :-)

Reply:
spis si nejsem jisty, jestli to nema mit jeste jednu dimenzi (~len? kdyz to ma zaznamenavat kontexty a vahy)

Reply:
čas je ta nula a je fakt že tam chybí `context_vector_size`... Jak je to řešený v tý konkatenaci?

Reply:
dokonce to bylo tak, že contextům chyběla vector size a váhám zase chyběla dýlka enkodéru. fixknul jsem to v poslendím komitu